src/app/shared/modules/create-product-modal/create-product-modal.component.spec.ts

```diff
@@ -313,5 +337,33 @@ describe('CreateProductModalComponent', () => {
         ).toBe('test test');
       });
     });
+
+    describe('Create new Location Btn', () => {
+      it('should add new  record to counts property - form', fakeAsync(() => {
+        fixture.detectChanges();
+        spyOn(component.dialogRef.componentInstance, 'addCountsRecord');
+        harness.addNewCountsRecordBtnClick();
+        tick();
+        fixture.detectChanges();
+
+        expect(
+          component.dialogRef.componentInstance.addCountsRecord
+        ).toHaveBeenCalled();
+      }));
+    });
+
+    describe('Remove location Btn', () => {
```

Comment:
This description splitting does not make big sense here, better remove those and just leave to ***it*** cases for simplicity.